### PR TITLE
Create initial manuscript data pipeline dag

### PR DESCRIPTION
**Pull request by seanwiseman**
A clean PR to get this in.

It currently goes all the way to generating the `CSV` files, the last step is getting `db_manager` to be callable within `airflow`.

After that is done, I will add in the final step in an additional PR.

### CODE REVIEW

File: airflow/.docker/entrypoint.sh [hunk 1]
```diff
@@ -59,12 +59,18 @@ wait_for_redis() {
   fi
 }
 
+install_csv_generator() {
+    # hack: will be removed once `csv_generator` is hosted on `pypi`
+    pip install --user csv-generator/dist/csv_generator-0.0.1-py3-none-any.whl
```
Comment (de-code) [inline airflow/.docker/entrypoint.sh]: I meant to ask where do we get that file from. The dev needs to run the setup?

Comment (seanwiseman) [reply airflow/.docker/entrypoint.sh]: Yes, (just adding the readme entry).

I am just putting it in a script at present (as this step will be removed at some point) and will add in the `db_manager` wheel file generation step to it once that's ready.

File: .env [hunk 2]
```diff
@@ -1,3 +1,7 @@
+ARCHIVE_BUCKET=fake-archive-bucket
+INCOMING_BUCKET=fake-bucket
+AWS_ACCESS_KEY_ID=
```
Comment (de-code) [inline .env]: I wonder whether in the AWS credentials case it would be better to mount `~/.aws/credentials`? (the `boto` library also supports that)

Comment (seanwiseman) [reply .env]: Yes that should work. I will test it out and include it if all goes well in the next pr 👍

**Pull request merged**